libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/CodegenSchema.kt

```diff
@@ -27,12 +28,14 @@ class CodegenSchema(
     val generateDataBuilders: Boolean,
 )
 
-internal class CodegenType(
+@ApolloInternal
+class CodegenType(
```

Comment:
Side effect of needing to have access to the fqcn of the input an enum types from the KSP processor to validate that they have the expected Kotlin type. Maybe there's a better way.